src/lib/y2firewall/clients/firewall.rb

```diff
@@ -40,16 +39,20 @@ def initialize
         textdomain "firewall"
       end
 
+      # TRANSLATORS: firewall-config and firewall-cmd are the name of software utilities,
+      # so they should not be translated.
+      NOT_SUPPORTED = N_("YaST currently does not have a module for configuring" \
```

Comment:
In general I suggest here to remove `\n` as for translators it is hard to place it properly to have reasonable line lenght and instead use where make sense later this method https://github.com/yast/yast-yast2/blob/5a2d13714e18c295f7e7973a23c4a872470c3bf4/library/general/src/lib/ui/text_helpers.rb#L33 on result of translation.